lib/types/api.types.ts

```diff
@@ -358,20 +370,9 @@ export type AppExtensionSDK = ConfigAppSDK
 /** @deprecated consider using {@link KnownAppSDK} */
 export type KnownSDK = KnownAppSDK
 
-export interface Locations {
-  LOCATION_ENTRY_FIELD: 'entry-field'
-  LOCATION_ENTRY_FIELD_SIDEBAR: 'entry-field-sidebar'
-  LOCATION_ENTRY_SIDEBAR: 'entry-sidebar'
-  LOCATION_DIALOG: 'dialog'
-  LOCATION_ENTRY_EDITOR: 'entry-editor'
-  LOCATION_PAGE: 'page'
-  LOCATION_HOME: 'home'
-  LOCATION_APP_CONFIG: 'app-config'
-}
-
 export interface ConnectMessage {
   id: string
-  location: Location[keyof Location]
+  location: Locations[keyof Locations]
```

Comment:
this was 100% a typo and typing bug...

Reply:
It's crazy that TS didn't notice that.

Reply:
yeah pretty glaring blind spot here, huh! 😱